cpp/meson.build

```diff
@@ -0,0 +1,19 @@
+project('Lift-Kata', 'cpp', default_options : ['cpp_std=c++17'])
+
+gtest_dep = dependency('gtest', main : false, required : false)
+gmock_dep = dependency('gmock', main : true, required : false)
```

Comment:
I'm not familiar with meson, so this is probably a red herring, but am curious as to whether this `true` creates a dependency on gmock, as I haven't noticed any uses of it...

Reply:
I guess that's a legacy from another kata that uses gmock and approval tests to stimulate the code. gmock isn't used in this code. As CLion was so dependent on CMake I didn't spend to much time with the meson builds.